src/services/services.ts

```diff
@@ -2051,6 +2051,18 @@ namespace ts {
             return true;
         }
 
+        function getAutoCloseTagAtPosition(fileName: string, position: number): string | undefined {

```

Comment:
i would make the return type an object with a `newText: string` property to allow us to add more info as we need in the future.